dev/jenkins/build.sh

```diff
@@ -20,6 +20,12 @@ then
   git clean -fdx
 fi
 
+local mvn_args=""
+if [ -n "${ALLUXIO_MVN_RUNTOEND}" ]
+then
+  mvn_args+=" -fn --fail-at-end"
```

Comment:
no need leading space in string

Reply:
it is not necessary, but this makes it easier to copy/paste later?

Reply:
true that, and there's no harm in having the extra space